app/controllers/HomeController.scala

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext}
+
+/**
+ * The controller for the index pages.
+ */
+class HomeController(cc: ControllerComponents)(implicit ec: ExecutionContext)  extends AbstractController(cc) {
+  def index() = Action { implicit request: Request[AnyContent] =>
```

Comment:
Does this need user authentication on it?

Reply:
This PR doesn't address authentication, but it probably should now that Typerighter is out in the wild. So -- yes! I'll take a look at some projects, are there any you'd consider to be good to steal from/wildly unidiomatic? (It might be nice to consider an approach that would permit other organisations to use the project, I remember you were doing something like this for the Grid)

Reply:
Ah ok, not a blocker if it doesn't already have auth.

I think Composer will be accessing the Typerighter API directly? If that's the case then the precedent would be to use Panda in [verify mode](https://github.com/guardian/pan-domain-authentication/#to-verify-logins).

However that does make it non-trivial for others to use Typerighter with authentication, as they need to set up Panda and [login.gutools](https://github.com/guardian/login.gutools). We've found that's quite a lot of work for people working on the Grid.

[google-play-auth](https://github.com/guardian/play-googleauth) (as used by Ophan) is slightly simpler to set up but wouldn't have the single-sign-on behaviour of Panda so using it for AJAX requests from Composer might not be appropriate as users couldn't interactively authenticated themselves.

Interesting, the Amazon ALB load balancer can [authenticate users directly](https://aws.amazon.com/blogs/aws/built-in-authentication-in-alb/). This could mean we push out authenticated as a concern from Typerighter itself, essentially making it pluggable for any users based on what AWS support. This woud be at the expense of (a bit) more expensive load balancer compared to the classic ones we are currently using in other tools.

Summarising my opinion:
 - Fastest to implement: [Panda in verify mode](https://github.com/guardian/pan-domain-authentication/#to-verify-logins)
 - Most interesting for the future (including others using Typerighter): [ALB authentication](https://aws.amazon.com/blogs/aws/built-in-authentication-in-alb/)

cc @sihil for his opinion

Reply:
@mbarton This is a really useful and detailed reply, thanks! An ALB-based solution seems like a step forward for projects more generally, and the [pricing doesn't look like it'll be significantly different with the usage we're expecting](https://aws.amazon.com/elasticloadbalancing/pricing/) (low hundreds of users, low tens concurrently). I'll take a look.

Reply:
Great! We have experimented with that strategy as well on Investigations so would be really interested to pair to get it to work